<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionViewBase</class>
 <widget class="QDialog" name="VersionViewBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>281</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>281</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>393</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="DeviceLabel">
       <property name="text">
        <string>Device:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DevVersionLabel">
       <property name="text">
        <string>Device Version:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SerialNrLabel">
       <property name="text">
        <string>Serial Number:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="AdjDataLabel">
       <property name="text">
        <string>AdjustmentData:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PCBVersionLabel">
       <property name="text">
        <string>PCB Version:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PCBServerLabel">
       <property name="text">
        <string>PCB Server:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DSPVersionLabel">
       <property name="text">
        <string>DSP Version:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DSPServerLabel">
       <property name="text">
        <string>DSP Server:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>197</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="DeviceText">
       <property name="text">
        <string>----------</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DeviceVersionText">
       <property name="text">
        <string>----------</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SerialNrText">
       <property name="text">
        <string>----------</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="AdjDataText">
       <property name="text">
        <string>----------</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PCBVersionText">
       <property name="text">
        <string>----------</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PCBServerText">
       <property name="text">
        <string>----------</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DSPVersionText">
       <property name="text">
        <string>----------</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DSPServerText">
       <property name="text">
        <string>----------</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
